subsys/net/lib/nrf_provisioning/src/nrf_provisioning_coap.c

```diff
@@ -511,6 +525,23 @@ static int send_response(struct coap_client *client,
 	return 0;
 }
 
+static void suspend_dtls_session(struct coap_client *client,
+				struct nrf_provisioning_coap_context *const coap_ctx)
+{
+	int ret;
+
+	/* Send a request to suspend DTLS session */
+	ret = send_coap_request(client, COAP_METHOD_POST, dtls_suspend, NULL, 0, coap_ctx, false);
+	if (ret < 0) {
+		LOG_WRN("Failed to send CoAP request");
+	}
+
+	LOG_DBG("Response code %d", coap_ctx->code);
+	if (coap_ctx->code == COAP_RESPONSE_CODE_NOT_FOUND) {
+		LOG_WRN("DTLS session suspension not supported");
+	}
```

Comment:
How do we get response to non-confirmable request?

Reply:
Server seems to send a response to non-confirmable request.